src/ctia/http/routes/crud.clj

```diff
@@ -43,7 +43,7 @@
            can-patch?
            can-search?
            can-get-by-external-id?]
-    :or {hide-delete? true
+    :or {hide-delete? false
```

Comment:
I don't know if this change should be part of this PR or could you add a comment in the PR's description?

Reply:
it's inherited from my previous branch, that won't hurt though